File: exercicios em C/q57.cpp
#include <stdio.h>

int main(void){
	int i, n, num1, soma;
	num1 = 1;
	soma = 1;
	printf("\nEste progrma determina o produtorio (multiplicacao) dos numeros inteiros no intervalo de 1 ate n");
	printf("\nDigite o valor de n: ");
	scanf("%d", &n);
	for (i = 1; i <= n; i++){
		if (i == n){
			printf(" %d", num1);
		}
		else{
			printf(" %d *", num1);
		}
		soma = soma * num1;
		num1++;
	}
	printf(" = %d.", soma);
	printf("\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q57.py
import os

num1 = 1
soma = 1
print("\nEste progrma determina o produtório (multiplicação) dos números inteiros no intervalo de 1 até n")
n = int(input("\nDigite o valor de n: "))
i = 1
while i <= n:
        print(" ",num1)
        soma = soma * num1
        num1 = num1 + 1
        i = i + 1

print(" = ",soma,"")
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")
